ksql-engine/src/main/java/io/confluent/ksql/util/QueryMetadata.java

```diff
@@ -95,6 +102,36 @@ public void close() {
     }
   }
 
+  private Set<String> getInternalTopicSet(
```

Comment:
Looks to be some duplication here, with the clean up code in `KafkaTopicClientImpl.deleteInternalTopics`, which also needs to get the list of internal topics.   These should probably share an implementation.